src/assets/styles/_base.scss

```diff
@@ -8,13 +8,16 @@ $palette-five: #481620;
 $default-padding: 5px;
 $default-horizontal-padding: 5px;
 $default-vertical-padding: 5px;
-$default-card-height: 150px;
-$default-card-width: 100px;
-$in-play-width: $default-card-width * 7; 
-$in-play-height: $default-card-height*2;
+$game-width: 1024px;
+$game-height: 768px;
+$card-height: 110px;
```

Comment:
$card-height and $card-width are used in multiple scss files, so have to be global